runtime/scripts/question.js

```diff
@@ -555,6 +558,7 @@ Question.prototype = /** @lends Numbas.Question.prototype */
             }
             q.signals.trigger('partsGenerated');
         });
+        this.events.trigger('loadFromJSON');
```

Comment:
this can only happen once, and there are already signals, so this is unnecessary